docs/UserGuide.md

```diff
@@ -227,18 +227,27 @@ The following subsections will elaborate on the specific details of each TutorBu
 
 Displays URL to TutorBuddy's User Guide and a list of all commands and their usages.
 
-![help message](images/helpMessage.png)
+![help message window](images/helpMessageWindow.png)
 
 Format: `help`
 
+To access the help page, do one of the following:
+1. type `help` in the command box 
+2. press <kbd>f1</kbd>
+3. click the 'Help' button on the top left and click on the dropdown
+![help message](images/helpMessage.png)
+
 ##### Listing all students and sessions: `list`
 
 Displays list of all students and all sessions in TutorBuddy.
 
-![list](images/command_list.png)
-
 Format: `list` <br>
 
+Example:
+1. Type in the command `list` and press <kbd>Enter</kbd>.
```

Comment:
@enhao25's enter is lowercase. Maybe y'all can standardize? I personally think this Enter is better.

Reply:
Can! I will update from enter to Enter also!